shotover-proxy/example-configs/cassandra-rewrite-peers/config.yaml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
What is this for? Dont we instruct the user to use the generic config.yaml?

Reply:
Needs to be mounted. The reason we can't use the base one is because then it won't mount on the user's machine. This means as long as they put it in the same directory it will work.

Reply:
ah, makes sense